### PR TITLE
Add first version of the deploy testnet node script; Cleanup setup_va…

### PR DESCRIPTION
**Pull request by ognjenkurtic**
First version of the instructions for runnning a testnet of baseledger-plateau

### CODE REVIEW

File: tests/setup-validators.sh [hunk 1]
```diff
@@ -71,9 +71,6 @@ do
 # move the genesis in
 docker cp ./genesis.json $VALIDATOR_CONTAINER_BASE_NAME$i:/validator/config/genesis.json
 
-ARGS="$BASELEDGER_HOME --keyring-backend test"
```
Comment (skosito) [inline tests/setup-validators.sh:74]: is this removed by mistake?

File: baseledger/ops/deploy_testnet_node.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+# Initialization
+
+Finspot node:
+
+1. Install latest golang
+2. Install cosmovisor: Follow the *deployment_and_chain_upgrade.md*
+3. Place binaries in the respective folders as described in *deployment_and_chain_upgrade.md*
+4. Initialize: cosmovisor init validator --chain-id=baseledger
+5. Configure genesis: Edit the genesis file for various params (voting time, tokens metadata etc.)
```
Comment (skosito) [inline baseledger/ops/deploy_testnet_node.md:19]: just to add, turn off inflation, unbonding time (at least for testing), min_self_delegation should be set to 1, not sure if this is genesis

File: baseledger/ops/deploy_testnet_node.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+6. Generate validator account: cosmovisor keys add --keyring-backend test validator (make sure to write down the address and the mnemonic)
+7. Generate orchestrator account: cosmovisor keys add --keyring-backend test orchestrator (make sure to write down the address and the mnemonic)
+8. Add faucet account with allocation: cosmovisor add-genesis-account --keyring-backend test baseledger1xgs5tamqre7rkz5q7d5fegjsdwufxxvt36w0a0 10000000000work
```
Comment (skosito) [inline baseledger/ops/deploy_testnet_node.md:22]: faucet should have stake as well

File: baseledger/ops/deploy_testnet_node.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+9. Add validator account with allocation: cosmovisor add-genesis-account --keyring-backend test <validator_address> 10000000000stake,10000000000work
```
Comment (skosito) [inline baseledger/ops/deploy_testnet_node.md:23]: should we give validator so much power? maybe 1 stake, 0 work?

File: baseledger/ops/deploy_testnet_node.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+10. Add orchestrator account with allocation: cosmovisor add-genesis-account --keyring-backend test <orchestrator_address> 1work
+11. Add gentx transaction: cosmovisor gentx --keyring-backend --moniker finspot_validator --ip <validator_ip_address> validator 10000000000stake
```
Comment (skosito) [inline baseledger/ops/deploy_testnet_node.md:25]: what is last argument here stake again?

File: baseledger/ops/deploy_testnet_node.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+12. Extract the genesis and add it to the BASELEDGER_PACKAGE
+
+Other nodes:
```
Comment (skosito) [inline baseledger/ops/deploy_testnet_node.md:28]: do we also need instruction for when node join after initial set?

**Review by skosito — APPROVED** [review]
Mostly comments and questions, please check them, but approved

File: tests/setup-validators.sh (hunk 1 above)
Comment (ognjenkurtic) [reply tests/setup-validators.sh:74]: nope, it is already set above

File: baseledger/ops/deploy_testnet_node.md (hunk 3 above)
Comment (ognjenkurtic) [reply baseledger/ops/deploy_testnet_node.md:22]: ok

File: baseledger/ops/deploy_testnet_node.md (hunk 2 above)
Comment (ognjenkurtic) [reply baseledger/ops/deploy_testnet_node.md:19]: ok, lets add during testing today

File: baseledger/ops/deploy_testnet_node.md (hunk 4 above)
Comment (ognjenkurtic) [reply baseledger/ops/deploy_testnet_node.md:23]: let's fix today during testing

File: baseledger/ops/deploy_testnet_node.md (hunk 6 above)
Comment (ognjenkurtic) [reply baseledger/ops/deploy_testnet_node.md:28]: yes, lets add after testing

File: baseledger/ops/deploy_testnet_node.md (hunk 5 above)
Comment (ognjenkurtic) [reply baseledger/ops/deploy_testnet_node.md:25]: self delegation of this validator

**Pull request merged**